src/api/.rubocop.yml

```diff
@@ -32,6 +32,10 @@ Style/AndOr:
 Style/AsciiComments:
   Enabled: true
 
+# This cop checks how the *when*s of a case expression are indented in relation to its `case` or `end` keyword.
```

Comment:
As I said before, I just wanted to highlight it somehow, regardless if this syntax is known for bold in markdown. I mean, I don't see the point in adjusting it since it's just a comment but well, I will do it.